README.md

```diff
@@ -110,13 +113,4 @@ def stats(message, start_date=None, end_date=None):
 ```
 
 
-And add the plugins module to `PLUGINS` list of mattermost_bot settings, e.g. mattermost_bot_settings.py:
```

Comment:
Why this section was removed? It's a very necessary for devops/programmers and etc.